mypy.ini

```diff
@@ -3,7 +3,7 @@ strict = True
 show_error_codes = True
 files = pulpcore/**/*.py
 
-[mypy-urllib3]
+[mypy-urllib3.*]
```

Comment:
What does this change do?

Reply:
It made the linting pass. I think it's needed for the import from urllib3.utils